helper/token/decode.go

```diff
@@ -0,0 +1,39 @@
+package token
```

Comment:
I suggest making this helper part of the sdk instead, since I imagine you want to make use of this outside of Vault.

Reply:
Yeah that's a good point. I'll rework the code